server/dns-service/dnsservice-dnsjava/src/test/java/org/apache/james/dnsservice/dnsjava/DNSJavaServiceTest.java

```diff
@@ -142,6 +145,7 @@ public void testMXCatches() throws Exception {
             res.add("");
             fail("MX Collection should not be modifiable");
         } catch (UnsupportedOperationException e) {
+            LOGGER.info("Ignored error", e);
```

Comment:
I can not as the exception thrown would prevent latest test instructions to be executed.